api/index.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const router = express.Router();
+const ctrlContact = require("../controller");
+
+router.get("/contacts", ctrlContact.get);
+router.get("/contacts/:id", ctrlContact.getById);
+router.post("/contacts", ctrlContact.create);
+router.put("/contacts/:id", ctrlContact.update);
+router.patch("/contacts/:id/status", ctrlContact.updateStatus);
```

Comment:
```suggestion
router.patch("/contacts/:contactId/favorite", ctrlContact.updateStatus);
```
Tak powinno być, zmień rasztę kodu dostosowując się do tego